src/mod_keystore.erl

```diff
@@ -84,7 +78,11 @@ config_spec() ->
         items = #{<<"ram_key_size">> => #option{type = integer,
                                                 validate = non_negative},
                   <<"keys">> => #list{items = keys_spec()}
```

Comment:
We could validate the keys here by just adding `format_items = map` and thus checking the uniqueness without the need for the whole `validate_key_ids` function.

Reply:
The keys would be validated for uniqueness, see https://github.com/esl/MongooseIM/blob/master/src/config/mongoose_config_parser_toml.erl#L190

Reply:
OK, thanks for the clarification - done.